### PR TITLE
Fix duplicate log messages

### PR DESCRIPTION
**Pull request by kamilkrzyskow**
MkDocs 1.5 started using f-strings when creating build time log messages, therefore each string is unique (as the time typically changes slightly). 
The old solution stopped working, so now instead I've created a different DuplicateFilter class, which looks at the prefix of the log and not the exact match.
Additionally this PR changes makes more logging changes:
- adds messages about default language being built
- scopes the lunr.js log warning only to the built language

### CODE REVIEW

File: mkdocs_static_i18n/plugin.py [hunk 1]
```diff
@@ -36,6 +37,10 @@ def on_config(self, config: MkDocsConfig):
         # first execution, setup defaults
         if self.current_language is None:
             self.current_language = self.default_language
+            log.info(
+                f"Building '{self.current_language}' documentation to directory: {config.site_dir}"
```
Comment (ultrabug) [inline mkdocs_static_i18n/plugin.py:41]: shouldn't we add 

```suggestion
                f"Building '{self.current_language}' documentation to directory: {config.site_dir}/{self.current_language}"
```

File: mkdocs_static_i18n/utils.py [hunk 2]
```diff
@@ -26,3 +27,17 @@ def get_plugin(name: str, config: MkDocsConfig) -> Optional[Plugin]:
             return p
 
     return None
+
+
+class i18nLoggingFilter:
```
Comment (ultrabug) [inline mkdocs_static_i18n/utils.py:32]:
```suggestion
class I18nLoggingFilter:
```

I'd rather classes start with a uppercase letter please

**Review by ultrabug — CHANGES_REQUESTED** [review]
Thanks a lot, almost there

File: mkdocs_static_i18n/plugin.py (hunk 1 above)
Comment (kamilkrzyskow) [reply mkdocs_static_i18n/plugin.py]: The `config.site_dir` uses `\` on Windows, also the "Building..." log message for the alternates was in the `on_page_build` event.
Based on your suggestion, I scoped the log message to the `on_config` event, and also add the proper path suffix for each language.

File: mkdocs_static_i18n/utils.py (hunk 2 above)
Comment (kamilkrzyskow) [reply mkdocs_static_i18n/utils.py]: Me too, I just forgot myself out of habit of writing `i18n` in that case. Good catch. I think Pylint detects those kinds of name issues, maybe it's worth adding it to the hatch style run 🤷

**Comment by ultrabug** [comment]
Thank you!

**Pull request merged**